src/js/components/sharedComponents/dropdowns/NewPicker.jsx

```diff
@@ -154,17 +154,25 @@ const NewPicker = ({
                             onClick: createOnClickFn(option.onClick)
                         }))
                         .map((option) => (
-                            <li key={uniqueId()} className={`filter__dropdown-list-item ${option?.classNames ? option.classNames : ''} ${option.name.trim() === selectedOption.trim() ? 'active' : ''}`}>
-                                <button
-                                    className="filter__dropdown-item"
-                                    value={`${option.value || option.name}`}
-                                    onClick={(e) => {
-                                        e.preventDefault();
+                            <button
```

Comment:
Maybe we should get on a call and talk about this. li should be the direct descendant of the ul (instead of the button)